src/benchmarklib/benchmark_runner.cpp

```diff
@@ -32,6 +33,9 @@ BenchmarkRunner::BenchmarkRunner(const BenchmarkConfig& config,
       _benchmark_item_runner(std::move(benchmark_item_runner)),
       _table_generator(std::move(table_generator)),
       _context(context) {
+  SQLPipelineBuilder::default_pqp_cache = std::make_shared<SQLPhysicalPlanCache>();
+  SQLPipelineBuilder::default_lqp_cache = std::make_shared<SQLLogicalPlanCache>();
```

Comment:
I'll need to move this once TPC-C comes in, but for the current master it's fine